# Add riscv64 to the list of machines that do not suppor -m64 flag
--- Makefile.orig	2025-01-16 15:13:46.238705778 +0100
+++ Makefile	2025-01-16 15:14:27.530581480 +0100
@@ -363,7 +363,7 @@
 endif
 
 ifeq ($(m64),yes)
-  ifeq (,$(filter $(MACHINE),ia64 alpha mips64 mips64el aarch64))
+  ifeq (,$(filter $(MACHINE),ia64 alpha mips64 mips64el aarch64 riscv64))
     GT_CFLAGS += -m64
     GT_LDFLAGS += -m64
     SQLITE_CFLAGS += -m64
